ios-task/README.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+This task is intended for people who interview for our iOS Software Engineer roles.
+
+Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting the following tasks.
+Take your time to polish your project. 
+Testing is very important for us, add tests where is possible and design the app in a way that should be easy to be tested.
```

Comment:
I wonder if we should be explicit here about 'unit tests'? Also we could mention that any test framework they are most comfortable with is welcome be it XCTest, Quick/Nimble?

Reply:
sure.
The more we are clear that we'd like to see unit tests the better.
What do you suggest as a rephrasing?

Reply:
How about 'At Conde Nast we unit test our code and strive to allow code design to be guided by these. Please add unit tests where possible in your solution. Feel free to use XCTest or if it's your preference a 3rd party utility like Quick/Nimble'

Reply:
updated